src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,55 @@
+.nav {
+  padding-right: 50px;
```

Comment:
make that a padding for your header, not for nav, you can clearly see that your test didn't pass because of that